### DISCUSSION TITLE
SPL and SPPT values?

### DISCUSSION THREAD

**Discussion by chege54** [seerge/g-helper#373] [Q&A] [answered]
Hi!
A long time ago when I  have tried Armory Crate I have seen under the OC/CPU tab there are two sliders.
One is related to SPL other is SPPT. According to this reddit page: https://www.reddit.com/r/overclocking/comments/liwjrx/spl_and_sppt_values_for_ryzen_5900hx/ 
SPL refers to the "Sustained Power Limit"
SPPT probably refers to the "Slow PPT power limit"

My question would be, If I change the "Power Limit (PPT)" via G-Helper which one is changed?
Thank you so much!

**Comment by seerge** [comment]
@chege54 it changes both :)

**Reply by chege54** [reply]
It sets both to the same value?
What about the upper limit? The sliders top seems to me a bit high

**Reply by seerge** [reply, marked as answer]
Yes, it sets same value.
As for slider range it’s universal to accommodate all possible cases. Your bios has hardcoded limits, so values higher than that won’t have extra effect

**Reply by chege54** [reply]
Thank you so much!
